GetTeched.Console.CodingTracker/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using coding_tracker;
+using System.Configuration;
+
+bool testMode = true;
+
+string connectionString = ConfigurationManager.AppSettings.Get("ConnectionString")!;
+DatabaseManager databaseManager = new();
+SessionController sessionController = new();
```

Comment:
In this file you get the `connectionString` and in the `SessionController` you also get the `connectionString`. File IO relatively takes a lot of time, so, for performance it would be nice to only load the `connectionString` once.

Reply:
I guess I could have combined  `DatabaseManger` and `SessionController` actually don't think it was necessary to split those, then I would only need to call `connectionString` once.